packages/apps/docs/src/components/Layout/Blog/BlogStyles.ts

```diff
@@ -0,0 +1,20 @@
+import { styled, StyledComponent } from '@kadena/react-components';
+
+export const ArticleTopMetadata: StyledComponent<'div'> = styled('div', {
```

Comment:
o..forgot. Can we use react-ui vanilla extract?
we need to refactor to that lib any way.
Only when using old stuff we still use react-components

Reply:
my bad